src/Game/Chess/Internal/HashConstants.hs

```diff
@@ -1,42 +1,7 @@
-{-# LANGUAGE PatternSynonyms #-}
```

Comment:
We discussed this a bit over email @mlang but to recap, circular dependencies prevent the Internal/Position logic from depending on Polyglot. Moving it here seemed simpler than duplicating the hashing logic, generating new hashes independent of Polyglot etc.